packages/engine/bin/cli/src/manifest.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
This file can be removed.

Reply:
I think that's a weird remnant of a rebase with main. Cheers for spotting

Reply:
yeah, probably.